openlibrary/templates/covers/add.html

```diff
@@ -59,14 +59,14 @@
 
             <div class="formElement">
                 <div class="label">
-                    <label id="imageWeb" for="imageUrl">$:_("<span class="highlight"><u>Or</u></span>, paste in the image URL if it's on the web.")</label>
+                    <label id="imageWeb" for="imageUrl">$:_('<span class="highlight"><u>Or</u></span>, paste in the image URL if it is on the web.')</label>
```

Comment:
This will let us avoid having to update the i18n files

```suggestion
                    <label id="imageWeb" for="imageUrl">$:_("<span class=\"highlight\"><u>Or</u></span>, paste in the image URL if it's on the web.")</label>
```